.travis.yml

```diff
@@ -1,17 +1,13 @@
-sudo: false
-dist: trusty
 language: rust
-
-before_install:
-  - cargo install mdbook --vers '0.0.25' --debug --force
```

Comment:
The build failed because this was removed and we ran into issues with the cache before when we first setup. The builds aren't that long that we really need to cache this. Can you please put this back in? :D

Reply:
Added `sudo` back now. I'm going to try to ape the rustc-guide setup as much as possible, if that's alright. (Consistency across rust projects is nice, no?) If that doesn't work still, we can revert to this, but I suspect the reason for this bug originally was the old version of *mdbook*.